app/components/tabbed-navigation.js

```diff
@@ -1,17 +1,23 @@
+import { tracked } from '@glimmer/tracking';
+import classic from 'ember-classic-decorator';
+import { classNames } from '@ember-decorators/component';
+import { action, computed } from '@ember/object';
 import $ from 'jquery';
 import Component from '@ember/component';
-import { computed } from '@ember/object';
 
-export default Component.extend({
-  classNames : ['tabbed-navigation'],
-  item       : null,
+@classic
+@classNames('tabbed-navigation')
+export default class TabbedNavigation extends Component {
+  @tracked item = null;
 
-  currentRoute: computed('session.currentRouteName', 'item', function() {
+  @computed('session.currentRouteName', 'item')
+  get currentRoute() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected getter 'currentRoute' to always return a value.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346)